cardano/src/config.rs

```diff
@@ -62,6 +63,21 @@ impl cbor_event::Deserialize for ProtocolMagic {
     }
 }
 
+pub type NetworkMagic = Option<i32>;
+
+impl From<ProtocolMagic> for NetworkMagic {
+    fn from(pm: ProtocolMagic) -> Self {
+        // FIXME: is there a better way to determine whether to emit
+        // NetworkMagic? There is a requiresNetworkMagic field in
+        // lib/configuration.yaml, but not in the genesis data.
+        if pm == ProtocolMagic::default() || *pm == 633343913 {
+            None
+        } else {
+            Some(*pm as i32)
+        }
+    }
+}
```

Comment:
I think it would be easier to put it, in the configuration of the CLI and hermes.